python/tk_desktop/project_commands_widget.py

```diff
@@ -162,7 +162,7 @@ def _text_for_item(self, item):
         if menu_name is None:
             return button_name
         else:
-            return "%s\n%s" % (button_name, menu_name)
+            return menu_name
```

Comment:
```
Maya
Maya 2017
```

now becomes
`Maya 2017`

I've also validated that this works with classic configs and apps like the publisher.